tests/ProductTest.php

```diff
@@ -18,14 +18,16 @@ public function testIsTrue()
             ->setIllustration('/public/uploads/5hCsZj27K.img')
             ->setDescription('Grand Theft Auto V est un jeu vidéo d\'action-aventure, développé par Rockstar North et édité par Rockstar Games.')
             ->setPrice(79.99)
-            ->setCategory($category);
+            ->setCategory($category)
+            ->setIsBest(1);
 
```

Comment:
Return an error because failed asserting.
For `setIsBest` and `getIsBest` replace `1` by `true`.